Valigator.Core/Valigator.Core.csproj

```diff
@@ -13,8 +13,4 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="Extensions\" />
```

Comment:
Was the folder removed on purpose?

Reply:
This was removed automatically because it isn't needed.